src/codelabs/async-await/index.md

```diff
@@ -0,0 +1,473 @@
+---
+title: "Asynchronous Dart codelab: futures, async, and await"
+description: A series of explanations and exercises for writing asynchronous code in Dart
+date: 2019-06-12
+tags: [ future, async, await, asynchronous, try, catch, error ]
```

Comment:
Sure, removed.

Reply:
It's really up to us how we use metadata for our static content. IMHO the out-of-the-box metadata jekyll exposes via [frontmatter](https://jekyllrb.com/docs/front-matter/) (categories, tags, date) is useful -- if not now, probably in the future. It's much easier to add as we publish each new article/page/codelab than all at once later if we decide we want it.

Reply:
I'm just worried about maintaining a bunch of stuff that doesn't get used. Or worse, that _is_ used, but not consistently, so that the results in whatever uses it are weird. E.g. would it be weird to have this page tagged "try" and "catch", but not have those tags in the language tour?

Reply:
I agree that, whatever we decide, we should be intentional about it. Perhaps this warrants creating an issue so we discuss it sooner rather than later.

Reply:
Please do.